lib/game.rb

```diff
@@ -64,9 +63,36 @@ def setup_game
 
   def display_boards
     puts "=============COMPUTER BOARD=============\n"
-    @users[:user2].board.render 
+    puts @users[:user2].board.render(true)
     puts "==============PLAYER BOARD==============\n"
-    @users[:user1].board.render 
+    puts @users[:user1].board.render(true)
   end
 
-end
\ No newline at end of file
+  def ship_placement 
+    ships = [["Cruiser", 3], ["Submarine", 2]]
+    ships.each do |ship|
+      puts "Enter the squares for the #{ship[0]} (#{ship[1]} spaces):"
+      input = gets.chomp
+      boat = Ship.new(ship[0], ship[1])
+      while !@users[:user1].board.valid_placement?(boat, input.split)
+        puts "Those are invalid coordinates. Please try again:"
+        input = gets.chomp
+      end
+      @users[:user1].place_ship(boat, input.split)
+    end
+  end
+
+  def ship_placement_computer 
+
+    ships = [["Cruiser", 3], ["Submarine", 2]]
+    ships.each do |ship|
+      boat = Ship.new(ship[0], ship[1])
+      coordinates = @users[:user2].board.random_coordinate_generator(rand(2), ship[1])
```

Comment:
Would we be able to have one method for ship placement instead of two separate methods?